localization/english/wc_traits_l_english.yml

```diff
@@ -0,0 +1,1027 @@
+﻿l_english:
+ trait_creature_human:0 "Human"
+ trait_creature_human_desc:0 "Many years ago, when the Titans left Azeroth, the old gods sent the Curse of the Flesh to the Titans' countless creations. The curse failed to bring them under the old gods' sway, but the vrykul - gloomy, cruel, and warlike giants of the north - were created to protect other servants of the Titans. Because of the curse, Vrykul women began to give birth to weak, ugly children. King Imiron, the ruler of the whole Vrykul people, found out about this and believed that the Titans turned away from his people. Thus, he renounced the creators, ordering the killing of the "spoiled" offspring - but not all mothers fulfilled the king's decree. Those who could not do this fled to the south, and their children became the ancestors of man. Humans are one of the youngest races in Azeroth, but they have already withstood many trials with honour. Proud and unbending, humans are diverse and capable of any job, but are most skilled at diplomacy and solving problems peacefully."
+ trait_creature_human_character_desc:0 "This character is a human. $trait_creature_human_desc$"
+ trait_creature_gnoll:0 "Gnoll"
+ trait_creature_gnoll_desc:0 "Gnolls are cunning, animal-like folk - a hybrid of man with hyena. A big hump, a long neck, and clumsy fingers do not stop them from fighting, and conflict itself is a way of life for this race. Gnolls are very violent creatures, even over the smallest dispute, they are liable to gnaw at each other's throats - and in a literal sense, as they devour all creatures, including members of their own race. In their society, the Cult of Power flourishes. Tribal leaders are among the strongest, while the weak are consumed. Gnolls are lazy and primitive, as they have not yet mastered metalworking and still prefer a nomadic way of life."
+ trait_creature_gnoll_character_desc:0 "$trait_creature_gnoll_desc$"
+ trait_creature_murloc:0 "Murloc"
+ trait_creature_murloc_desc:0 "Murlocs are a strange combination of humanoid and fish, an ancient two-legged race of amphibians. They live along the coasts and within the depts of oceans, the coasts of lakes and rivers, as well as in underwater ruins. Murlocs are capable of breathing on land, although they rarely risk moving too far from water. Their hunched appearance is rather awkward, but don't let this mislead you - some murlocs travel overland with great speed and agility. Murlocs do not care about individuality or creativity, as this does not help the survival of the tribe or most of their race. They value obedience above all else. Agility and cruelty are also important to them, reflected in their hunting skills and strength."
+ trait_creature_murloc_character_desc:0 "$trait_creature_murloc_desc$"
```

Comment:
Why do we have "This character is a human", but not in the other trait descriptions?

Reply:
> Why do we have "This character is a human", but not in the other trait descriptions?

`_character_desc` shows when the trait is on character. `_desc` shows when the trait is in an event option for example.

Reply:
Wouldn't it make more sense to put "This character is a ..." in all "trait_creature_..._character_desc" lines?

Reply:
Nvm, forget my previous question, I got it the wrong way.